### PR TITLE
CreatePortForwardingRuleCmd: Fix typo in description

### PR DESCRIPTION
**Pull request by resmo**

### CODE REVIEW

File: api/src/org/apache/cloudstack/api/command/user/firewall/CreatePortForwardingRuleCmd.java [hunk 1]
```diff
@@ -111,7 +111,7 @@
     @Parameter(name = ApiConstants.CIDR_LIST, type = CommandType.LIST, collectionType = CommandType.STRING, description = "the cidr list to forward traffic from")
     private List<String> cidrlist;
 
-    @Parameter(name = ApiConstants.OPEN_FIREWALL, type = CommandType.BOOLEAN, description = "if true, firewall rule for source/end pubic port is automatically created; "
+    @Parameter(name = ApiConstants.OPEN_FIREWALL, type = CommandType.BOOLEAN, description = "if true, firewall rule for source/end public port is automatically created; "
         + "if false - firewall rule has to be created explicitely. If not specified 1) defaulted to false when PF"
```
Comment (runseb) [inline api/src/org/apache/cloudstack/api/command/user/firewall/CreatePortForwardingRuleCmd.java]: actually, there is another one here 'explicitly' can you patch and squash ?

**Comment by resmo** [comment]
@runseb ah yes, sure!

**Comment by resmo** [comment]
Did some more text stylings. :)

**Comment by runseb** [comment]
thanks for the patch, keep them coming

**Pull request closed without merging**